client/src/api/schema/schema.ts

```diff
@@ -13114,6 +13140,12 @@ export interface operations {
                     "application/json": components["schemas"]["DatasetSummaryList"];
                 };
             };
+            /** @description Bad Request */
+            400: {
+                content: {
+                    "application/json": components["schemas"]["MessageExceptionModel"];
+                };
+            };
             /** @description Validation Error */
             422: {
                 content: {
```

Comment:
It seems the 422 is still getting generated in the resulting schema so `remove_422` doesn't seem to work as expected. We could keep the 422 as a validation-specific error though given the response data is in the format we want and adapt the client error handling to support it. What do others think about this?

Have a look at https://fastapi.tiangolo.com/tutorial/handling-errors/#override-request-validation-exceptions and this discussion https://github.com/tiangolo/fastapi/discussions/9061 if you haven't already, maybe there is an interesting solution there.